cognite/powerops/client/shop_result_files.py

```diff
@@ -121,6 +120,7 @@ def find_time_series(
         matches_attribute_name="",
     ) -> list[str]:
         """Find time series in the results file. Some keys are parsed as numbers"""
+        # TODO: accept either a single string or a list of strings
```

Comment:
Something that can be fixed now, or maybe create a Jira task instead to avoid having TODOs in the SDK(?) Just a thought @doctrino wdyt?

Reply:
This is a one liner, so yes please fix before merging. It is not worth a Jira issue.